BiweeklyMeetings/2022-07-05.md

```diff
@@ -0,0 +1,274 @@
+    + [CIP-0050? | Shelleys Voltaire decentralization update weekly update](#pr242)     
+  * [Discussions](#discussions)
+  * [Issues](#issues)
+  * [Close](#close)
+
+## Summary
+
+Rough transcript of 05/07/22 Editors CIP meeting, to increase transparency and dialogue with the community regarding proposed changes, implementations and considerations.  
+<sub>_Transcript might contain errors or miss pieces - call out issues as needed_
+</sub>  
+Editors meetings are [public](https://discord.gg/36ezQYTvgk), [recorded](https://www.youtube.com/c/CardanoFoundation/videos) and [transcript](https://github.com/cardano-foundation/CIPs/tree/master/BiweeklyMeetings): do join and participate for discussions/PRs of significance to you.  
+
+
+## Editors Meeting Flow
+
+**Triage/Review**: Some CIPs might fall out of grace or not get updated, a CIP that hasn’t seen activity for 3 months should be checked on, and appropriate action taken. Ex: did any of the recent changes obsolete current CIPs? Consider ‘Active’ -> ‘Obsolete’ transitions..  
+
+**Last Check**: Review of the PRIOR meetings Decisions  - if no objection, apply change (effectively a two week lag from decision to action, as a grace period)  
+
+**New CIPs Review**: CIPs up for review should be looked over collectively, with discussion where needed. (on top of the asynchronous reviews)  
+
+PR -> ‘Draft’: Needs format + approval.  
+‘Draft’ -> ‘Proposed’: Needs a PLAN towards Active + implementation.  
+‘Proposed’ -> ‘Active’:  Objective criteria as laid out observed, and consensus agreeing.   
+**Current Discussions**: What the current CIPs discussions are on social media / forums / Discord.  
+
+**Issues**: Review of the open Issues on the CIP GitHub 
+
+**Close**: Recap of actions taken and decisions. List the CIPs that are due for review.  
+
+
+## July 05 2022 Transcript
+**Attending Editors**: Matthias Benkort, Sebastien Guillemot, ~Frederic Johnson~, ~Robert Phair~. 
+**Guests**:
+
+### Triage
+
+#### PR274
+[PR274 CIP-0059? | Terminology Surrounding Core Features](https://github.com/cardano-foundation/CIPs/pull/274)
+
+**Matthias** - :  Yeah, we can start already and go right in the first item, right, CIP 59, which was proposed by Jared, I think after a chat on a different CIP or different debates around Vasil versus Daedalus, and what do names mean exactly in Cardano. A lot of people seem to have gotten confused by the names being employed. So he wrote a small, informational CIP, there is not really much to debate in a way except maybe if people disagree with... I know the similar thing given to that and they want to change it, but since there is a type history already. It just explains where the name comes from, what they are and the rationale behind is why they were people like that. Like between era, and a phase, and a network and protocol. And he has also put up a nice table which summarizes all of that, which is nice, so it gives just a bit more clarity about naming.
```

Comment:
```suggestion
**Matthias** - :  Yeah, we can start already and go right in the first item, right, CIP 59, which was proposed by Jared, I think after a chat on a different CIP or different debates around Vasil versus Babbage, and what do names mean exactly in Cardano. A lot of people seem to have gotten confused by the names being employed. So he wrote a small, informational CIP, there is not really much to debate in a way except maybe if people disagree with it and want to change it, but since there is a track record already... It just explains where the name come from, what they are and the rationale behind is, why they were chosen like that. Like between era, and a phase, and a network and protocol. And he has also put up a nice table which summarizes all of that, which is nice, so it gives just a bit more clarity about naming.
```